lib/ruby_lsp/requests/signature_help.rb

```diff
@@ -50,22 +51,12 @@ def provider
       end
       def initialize(document, index, position, context, dispatcher)
         super()
-        current_signature = context && context[:activeSignatureHelp]
         target, parent, nesting = document.locate_node(
-          { line: position[:line], character: position[:character] - 2 },
+          { line: position[:line], character: position[:character] },
```

Comment:
Just curious: what was the original `-2` for and why don't we need it anymore?

Reply:
It was basically trying to point the location at the method call when it was pointing to the empty space inside the parenthesis.

```ruby
foo( )
#   ^ if it pointed here
# ^ we brought it back here to find the call node
```

With the nested checks, we shouldn't need it anymore.